tests/test_async_methods.py

```diff
@@ -151,6 +151,39 @@ async def test_async_fetch_cosmetics_cars(api_key: str, response_flags: fn_api.R
 
     for cosmetic in cosmetics_cars:
         assert isinstance(cosmetic, fn_api.CosmeticCar)
+        _test_cosmetic_car(cosmetic)
+
+
+def _test_cosmetic_car(cosmetic: fn_api.CosmeticCar[Any]):
```

Comment:
The tests should probably assert types in them.
```suggestion
    for cosmetic in cosmetics_cars:
        _test_cosmetic_car(cosmetic)


def _test_cosmetic_car(cosmetic: fn_api.CosmeticCar[Any]):
    assert isinstance(cosmetic, fn_api.CosmeticCar)
```

This goes for all other tests as well.